lib/main_containers.pm

```diff
@@ -74,21 +79,16 @@ sub load_tests_docker_common {
 sub load_container_tests {
     my $runtime = get_required_var('CONTAINER_RUNTIME');
     boot_hdd_image;
-    # Container Image tests
-    if (get_var('FLAVOR', '') =~ /Container-Image/) {
+
+    if (is_container_image_test()) {
+        # Container Image tests
         loadtest 'containers/host_configuration' unless get_var("HDD_1") =~ /res/;
```

Comment:
`res`?

Reply:
RES=RedHat Expanded Support (https://openqa.suse.de/tests/7208229).
This could be avoided also by having some way to get the host OS (same problem as for container_diff).

Reply:
`res` sounds overly generic, it would also match `result` or other words... Should contain some delimiter and preferably a comment